code/datums/mind.dm

```diff
@@ -1215,6 +1226,42 @@
 				log_admin("[key_name(usr)] has automatically forged objectives for [key_name(current)]")
 				message_admins("[key_name_admin(usr)] has automatically forged objectives for [key_name_admin(current)]")
 
+	else if(href_list["brother"])
+		switch(href_list["brother"])
+			if("clear")
+				if(has_antag_datum(/datum/antagonist/brother))
+					to_chat("<span class='warning'><font size=3><b>You have been brainwashed! You are no longer a blood brother!</b></font></span>")
+					remove_antag_datum(/datum/antagonist/brother)
+					log_admin("[key_name(usr)] has de-brothered [key_name(current)]")
+					message_admins("[key_name_admin(usr)] has de-brothered [key_name_admin(current)]")
+
+			if("brother")
+				if(!has_antag_datum(/datum/antagonist/brother))
+					var/list/candidates = list()
+					for(var/mob/living/L in GLOB.alive_mob_list)
+						if(!L.mind || L.mind == current || L.mind.has_antag_datum(/datum/antagonist/brother))
+							continue
```

Comment:
You may want to include a check that makes sure the person has their brother preference on, so admins are only getting a list of people who want to play brother. Or I suppose you could leave it up to admins to choose.

Reply:
I'll leave it up to the admin, since they need to ask two players.  so having brothers turned on is a mute point.